.github/workflows/continuous-integration-workflow.yml

```diff
@@ -47,6 +47,9 @@ jobs:
       - name: maybe_disable_death_tests
         if: ${{ matrix.distro == 'fedora:rawhide' }}
         run: echo "GTEST_FILTER=-*DeathTest*" >> $GITHUB_ENV
+      - name: maybe_use_external_gtest
```

Comment:
With this change, Ubuntu jobs will use installed version of GTest instead of the internal one.

I have considered modifying `ci-containers` instead, but in the end I'm leaning towards leaving it like that. Any thoughts?

Reply:
IMHO, it's fine. It doesn't take long to install, right?

Reply:
Yep, below 10 seconds in the last couple of builds that I've checked.